subproviders/src/subproviders/metamask_subprovider.ts

```diff
@@ -56,16 +60,32 @@ export class MetamaskSubprovider extends Subprovider {
                     end(err);
                 }
                 return;
-            case 'eth_sendTransaction':
-                const [txParams] = payload.params;
+            case 'eth_call': {
```

Comment:
This was from legacy where you would only use Metamask for the signing component and provide your own RPC provider to do everything else.

Back then during ICO days Metamask's connection would be down 90% of the time 😂